circuits/cpp/barretenberg/cpp/src/barretenberg/honk/pcs/gemini/gemini.cpp

```diff
@@ -186,136 +186,7 @@ ProverOutput<Curve> GeminiProver_<Curve>::compute_fold_polynomial_evaluations(st
     return { fold_poly_opening_pairs, std::move(fold_polynomials) };
 };
 
```

Comment:
I've moved the definitions of the the GeminiVerifier methods back to the header. This avoids a lot of pain, at least while we need to instantiate it with both native and stdlib `Curve`s. It should be easy to revert once this is no longer the case (see bberg issue #673)